### DISCUSSION TITLE
ERROR: Failed building wheel for twisted

### DISCUSSION THREAD

**Discussion by sirthom9123** [django/channels#1530] [Q&A] [answered]
Hi guys, I keep encountering this error when I try to install channel. "ERROR: Failed building wheel for twisted" 
IDE: vscode
Python 3.8
Django 3
Windows 10
Please assisst.

**Comment by carltongibson** [comment, marked as answer]
Please don't post the same thing on multiple lists. As I said on django-developers: 

> This is an twisted issue (as per the error message)
> See https://github.com/django/channels/issues/1520 for more details/links.

**Reply by sirthom9123** [reply]
Thanks you can close the issue. I wasn't aware they're linked
